library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.cxx

```diff
@@ -58,16 +58,20 @@ void vtkF3DRendererWithColoring::SetPointSize(double pointSize)
   if (this->PointGaussianMapper)
   {
     vtkDataSet* dataset = this->PolyDataMapper->GetInput();
-    double bounds[6];
-    dataset->GetBounds(bounds);
-    vtkBoundingBox bbox(bounds);
 
-    double gaussianPointSize = 1.0;
-    if (bbox.IsValid())
+    if (dataset)
```

Comment:
@mwestphal this change is necessary to avoid a crash I was experiencing if the object failed to load for strange reason on my machine. It's not required, but it does not hurt to keep it